sample/src/main/kotlin/com/uber/autodispose/sample/ArchComponentActivity.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2017. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.uber.autodispose.sample
+
+import android.arch.lifecycle.ViewModelProviders
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.util.Log
+import android.widget.TextView
+import com.uber.autodispose.android.lifecycle.AndroidLifecycleScopeProvider
+import com.uber.autodispose.kotlin.autoDisposable
+import com.uber.autodispose.recipes.subscribeBy
+import io.reactivex.android.schedulers.AndroidSchedulers
+
+class ArchComponentActivity: AppCompatActivity() {
+
+  // Can be reused
```

Comment:
Can you flesh this out a bit regarding why it's safe?

Reply:
Not sure I understand. I just borrowed the `scopeProvider` definition from the `KotlinActivity` :)